app/assets/javascripts/hbx_admin.js

```diff
@@ -0,0 +1,21 @@
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
@vkghub A nitpick, can you rename the file to `app/assets/javascripts/time_jump.js`?